src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -325,10 +337,13 @@ void WindowMoveHandlerPrivate::MoveSizeEnd(HWND window, POINT const& ptScreen, c
         auto zoneWindow = std::move(m_zoneWindowMoveSize);
         ResetWindowTransparency();
 
-        auto windowInfo = FancyZonesUtils::GetFancyZonesWindowInfo(window);
+        bool hasNoVisibleOwnoer = FancyZonesUtils::HasNoVisibleOwner(window);
+        bool isStandardWindow = FancyZonesUtils::IsStandardWindow(window);
+        auto windowState = get_window_state(window);
 
-        if (windowInfo.standardWindow == false && windowInfo.noVisibleOwner == false &&
-          m_moveSizeStartWindowInfo.standardWindow == true && m_moveSizeStartWindowInfo.noVisibleOwner == true)
+        if ((isStandardWindow == false && hasNoVisibleOwnoer == false &&
+             m_moveSizeWindowInfo.standardWindow == true && m_moveSizeWindowInfo.noVisibleOwner == true) ||
+            (m_moveSizeWindowInfo.windowState != WindowState::MAXIMIZED && windowState == WindowState::MAXIMIZED))
         {
             // Abort the zoning, this is a Chromium based tab that is merged back with an existing window

```

Comment:
Need to update the comment to include the maximized case:
// or if the window is maximized by Windows when the cursor hits the screen top border

Reply:
Right, forgot that . Thanks! Fixed